core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/extensions.scala

```diff
@@ -170,6 +171,11 @@ case class Options(
   override def dataType: DataType = UnresolvedType
 }
 
+case class WithOptions(input: LogicalPlan, options: Expression) extends UnaryNode {
```

Comment:
it's missing a rule (or generator). rule can simply be stripping these off (or converting into a comment node), otherwise they won't get generated.

Reply:
Right. I had not decided what to do with them to be honest. I think that a Rule is probably best for these as the TSQL OPTIONS are highly unlikely to be useful. Table Hints may be a lot more useful.

Reply:
WithOptions now generated as comments preceding the MERGE (or whatever other statement receives OPTION() clause